skyrim-platform/src/platform_se/skyrim_platform/SkyrimPlatform.h

```diff
@@ -13,6 +16,15 @@ class SkyrimPlatform
   void AddTickTask(const std::function<void()>& f);
   void AddUpdateTask(const std::function<void()>& f);
   void PushAndWait(const std::function<void(int)>& task);
+  void Push(const std::function<void(int)>& task);
+  void Pull(RE::BSTSmartPointer<RE::BSScript::IFunction> fPtr,
```

Comment:
`Pull` and `Push` feel like antonyms here but they are not. It's confusing.
Please rename `Pull` to `ExecuteInGameThread` or something like this, more insightful for code readers.

Reply:
Renamed